src/main.js

```diff
@@ -13,10 +13,21 @@ const config = {
   height: 600,
   scene: [
     BootScene,
-    BootMenu,
-    MushroomScene,
     PlayerScene
   ]
 };
 
-new Phaser.Game(config);
+const login = $('#login');
+const chat = $('#chat');
+const game = $('#game');
+game.hide();
+chat.hide();
+login.show();
+
+$('#startButton').on('click', () => {
+  login.hide();
+  chat.show();
+  game.show();
+  new Chat($('#userNameInput')[0].value);

```

Comment:
Use the `get()` function to get the first element from the query, and use the `val()` to get the value of the input. With this way, I think the `get(0)` wont be necessary.